src/python/library/tritonclient/utils/shared_memory/shared_memory_handle.h

```diff
@@ -30,6 +30,21 @@
 #include <cuda_runtime_api.h>
 #endif  // TRITON_ENABLE_GPU
 
+#ifdef _WIN32
+#include <windows.h>
+struct SharedMemoryHandle {
+  std::string triton_shm_name_;
+  std::string shm_key_;
+#ifdef TRITON_ENABLE_GPU
```

Comment:
Yes, the block would currently always evaluate to false. However, because my plan was to begin work supporting GPU tensors next, I was adding all GPU structure now to avoid to having to track it down for the next PR. I can back out non-essential references to GPU in this PR.

Reply:
Ended up being removed entirely because it is no longer used.